client/lib/components/recent_numbers_graph.dart

```diff
@@ -40,6 +54,19 @@ class _RecentNumbersBarGraphState extends State<RecentNumbersBarGraph> {
           show: false,
         ),
         barTouchData: BarTouchData(
+          touchCallback: (touchResponse) {
+            setState(() {
+              swapDuration = Duration(microseconds: 1);
+              if (touchResponse.spot != null &&
+                  touchResponse.touchInput is! FlPanEnd &&
+                  touchResponse.touchInput is! FlLongPressEnd) {
+                selectedIndex = touchResponse.spot.touchedRodDataIndex;
+              } else {
+                selectedIndex = null;
+                swapDuration = Duration(milliseconds: 750);
```

Comment:
Should be using default and not need adjustment

Reply:
The duration for animating the swap between **Daily** and **Total Stats** is 750 milliseconds, however the animation for changing the bars should be instantaneous. This is here so after the bar color changes are done, the animation returns to its original state.